Makefile

```diff
@@ -1,58 +1,58 @@
 contentpack: pex
 	mkdir -p out/
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite en 0.17 --out=out/en.zip
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite en 0.16 --out=out/en.zip
```

Comment:
I am thinking creating another `PR` upgrading the version of the contentpack-maker.